public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx

```diff
@@ -3,19 +3,35 @@ import React from 'react';
 
 import { LokiOptionFieldsProps, LokiOptionFields } from './LokiOptionFields';
 
+interface Props extends LokiOptionFieldsProps {
+  queryType: string;
+  lineLimitValue: string;
+  onLineLimitChange: () => void;
+  onQueryTypeChange: () => void;
+  onKeyDownFunc: () => void;
+}
+
 const setup = (propOverrides?: LokiOptionFieldsProps) => {
```

Comment:
This `propOverrides` is not used at all in this test, so it would be preferable to remove it.